review previous portuguese docs (#17)
stack-spot/graphene-java-api-stack: metrics-plugin/docs/implementation.md

> To know the options of configuration and use of the capacities added for this plugin, see the documentation of [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator).

Reviewer: **Sugestão**
To know the options of configuration and **make** use of the capacities added for this plugin, see the documentation of [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator).

> To know the options of configuration and use of the capacities added for this plugin, see the documentation of [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator).

Reviewer 2: ```suggestion
For more details about the configuration options and how to use the capacities added in this plugin, see the [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator) documentation.

```
